react-app/src/components/PasswordAvatar.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react'
+import RoundedAvatar from './RoundedAvatar'
+import BackArrowImg from '../assets/back-arrow.svg'
+import ShowPasswordImg from '../assets/show-password.svg'
```

Comment:
Right now these imports give you paths to the image, not components, so I would recommend using the lower case (otherwise it's a little bit confusing).

Alternatively, you can use SVGO's feature, [supported by the CRA](https://create-react-app.dev/docs/adding-images-fonts-and-files/#adding-svgs), to import SVGs as real react components:
```js
import { ReactComponent as Logo } from './logo.svg';

function App() {
  return (
    <div>
      {/* Logo is an actual React component */}
      <Logo />
    </div>
  );
}
```